packages/core/src/order/order-selector.spec.ts

```diff
@@ -75,15 +75,15 @@ describe('OrderSelector', () => {
 
                 orderSelector = createOrderSelector(orderState, selectors.billingAddress, selectors.coupons);
 
-                expect(orderSelector.getPaymentId('some-provider-id')).toEqual('abc');
+                expect(orderSelector.getOrderPaymentId('some-provider-id')).toEqual('abc');
```

Comment:
I changed the name because I wanted to add it to `PaymentIntegrationSelectors`, while it's conflicting with another method there.